### PR TITLE
fix(overlay): hide 'configure settings' when features are disabled (Issue #2728)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

'Configure settings' and 'Change agent' are available in Overlay though all the features are disabled
Seems that EmptyChatSettings = 'empty-chat-settings' doesn't work at all? need to re-check for the whole DIAL

Issues:

- Issue #2728

**UI changes**

Снимок экрана 2024-12-10 в 16 35 32

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2769.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by IlyaBondar — APPROVED** [review]

File: apps/chat/src/components/Chat/EmptyChatDescription.tsx [hunk 1]
```diff
@@ -48,6 +53,12 @@ const EmptyChatDescriptionView = ({
   );
   const models = useAppSelector(ModelsSelectors.selectModels);
   const isIsolatedView = useAppSelector(SettingsSelectors.selectIsIsolatedView);
+  const enabledFeatures = useAppSelector(
+    SettingsSelectors.selectEnabledFeatures,
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/EmptyChatDescription.tsx]: we can use `isFeatureEnabled` instead

File: apps/chat/src/components/Chat/EmptyChatDescription.tsx [hunk 2]
```diff
@@ -48,6 +53,12 @@ const EmptyChatDescriptionView = ({
...
+  );
+  const isExternal = useAppSelector(
+    ConversationsSelectors.selectAreSelectedConversationsExternal,
+  );
```
Comment (IlyaBondar) [inline apps/chat/src/components/Chat/EmptyChatDescription.tsx]:
```suggestion
  const isExternal = isEntityIdExternal(conversation);
```

**Review by IlyaBondar — CHANGES_REQUESTED** [review]

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2769.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by nartovm** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2769.nightly-test.deltixhub.io
>E2E tests status: success

**Review by irinakartun — DISMISSED** [review]

**Review by irinakartun — APPROVED** [review]

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**